### PR TITLE
Add initial api spec

### PR DESCRIPTION
**Pull request by tonistuulNG**

### CODE REVIEW

File: docs/technical/api-spec.yml [hunk 1]
```diff
@@ -0,0 +1,110 @@
+openapi: 3.0.3
+info:
+  title: Distributed Message Room API
+  version: '1.0'
+  license:
+    name: MIT-0
+    url: 'https://opensource.org/licenses/MIT'
+  description: A well-orchestrated high-performance dumb pipe acting as a proxy pass.
```
Comment (turnerrainer) [inline docs/technical/api-spec.yml]: "A well-orchestrated high-performance proxy pass."

**Review by martinkearn — APPROVED** [review]

**Review by turnerrainer — CHANGES_REQUESTED** [review]
Please change the description as suggested

File: docs/technical/api-spec.yml (hunk 1 above)
Comment (joonasroosalung) [inline docs/technical/api-spec.yml]:
```suggestion
  description: A well-orchestrated high-performance proxy pass.
```

**Review by joonasroosalung — CHANGES_REQUESTED** [review]
Gotta update that description!

**Review by turnerrainer — APPROVED** [review]

**Review by joonasroosalung — APPROVED** [review]

**Pull request merged**